backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/scheduling/utils/VdsCpuUnitPinningHelper.java

```diff
@@ -188,6 +188,9 @@ public int countTakenCores(VDS host) {
         int numOfTakenCores = 0;
         for (int socket = 0; socket < host.getCpuSockets(); socket++) {
             for (int core = 0; core < hostCoresPerSocket; core++) {
+                if (getCpusInCore(getCoresInSocket(cpuTopology, socket), core).isEmpty()) {
+                    continue;
+                }
```

Comment:
Could you please add TODO to revise that?

Reply:
Done